.npmignore

```diff
@@ -1,39 +0,0 @@
-example/
```

Comment:
why remove this?

Reply:
I fixed to use `files`, we can remove the `.npmignore`.

https://docs.npmjs.com/files/package.json#files

https://github.com/react-hook-form/resolvers/pull/11/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13-R15

Reply:
can you do the same for react-hook-form?

Reply:
yea! I will fix 👍